frontend/src/app/pages/exercises/exercise/shared/exercise-map/shared/cater-capacity/cater-capacity.component.html

```diff
@@ -0,0 +1,10 @@
+<div *ngIf="canCaterFor" style="max-width: 20vw">
+    <span *ngIf="type === 'material'">Dieses Material </span>
+    <span *ngIf="type === 'personnel'">Diese Einsatzkraft </span>
+
+    <span *ngIf="canCaterFor.logicalOperator === 'or'">kann nur Patienten einer Verletzungsschwere gleichzeitig helfen.</span>
+    <span *ngIf="canCaterFor.logicalOperator === 'and'">kann Patienten verschiedener Verletzungsschweren helfen und dabei statt schwerer auch leichter verletzten helfen.</span>
+    <br><app-patient-status-badge [status]="'red'"></app-patient-status-badge>
+    <br><app-patient-status-badge [status]="'yellow'"></app-patient-status-badge>
+    <br><app-patient-status-badge [status]="'green'"></app-patient-status-badge>
```

Comment:
I would just write the numbers behind like this:

    
```suggestion
<p *ngFor="let status of ['red', 'yellow', 'green']">
    <app-patient-status-badge [status]="'red'"></app-patient-status-badge>:  {{ canCaterFor[status] }}
</p>
```

If you can't do `['red', 'yellow', 'green']` in the template then make a property in the component like:

```ts
public const caterForStatuses: (keyof CanCaterFor)[] = ['red', 'yellow', 'green'];
```

Reply:
Jup, that works, thank you ^^
Btw: That commit was marked as WIP, so this part has been refactored to use the css grid system. :)